### PR TITLE
AmongUs Edited

### PR DESCRIPTION
**Pull request by HydroGames-dev**
:+1: It's not yet tested

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
-name: AmongUs
+name: AmongUs-Beta
```
Comment (alvin0319) [inline plugin.yml]: This plugin is not beta and has never been beta.

File: plugin.yml [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 author: alvin0319
 main: alvin0319\AmongUs\AmongUs
 version: 0.0.10
-api: 3.0.0
-
-mcpe-protocol: 408
+api:
+- 2.0.0
+- 3.0.0
+- 4.0.0
```
Comment (alvin0319) [inline plugin.yml]: This is not for PM4, and also not for 2.0.0

File: plugin.yml [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+website: https://discord.gg/Py2vSwg3B3
```
Comment (alvin0319) [inline plugin.yml]: Website is https://github.com/alvin0319/AmongUs

**Comment by HydroGames-dev** [comment]
Oof

File: src/alvin0319/AmongUs/game/Game.php [hunk 4]
```diff
@@ -202,7 +202,7 @@ public function setMapItem(FilledMap $map) : void{
 	public function addPlayer(Player $player) : void{
 		if(!$this->hasPlayer($player)){
 			$this->players[] = $player->getName();
-			$this->broadcastMessage("Player " . $player->getName() . " has joined the game.");
+			$this->broadcastMessage(AmongUs::$prefix . "§e" . $player->getName() . " has joined the game.");
```
Comment (alvin0319) [inline src/alvin0319/AmongUs/game/Game.php]: Game::broadcastMessage has its own prefix, do not use AmongUs::$prefix

File: src/alvin0319/AmongUs/game/Game.php [hunk 5]
```diff
@@ -224,7 +224,7 @@ public function removePlayer(Player $player) : void{
 				unset($this->dead[array_search($player->getName(), $this->dead)]);
 				$this->dead = array_values($this->dead);
 			}
-			$this->broadcastMessage("Player " . $player->getName() . " has left the game.");
+			$this->broadcastMessage(AmongUs::$prefix . "§e" . $player->getName() . " has left the game.");
```
Comment (alvin0319) [inline src/alvin0319/AmongUs/game/Game.php]: too

**Review by alvin0319 — CHANGES_REQUESTED** [review]

File: src/alvin0319/AmongUs/game/Game.php (hunk 4 above)
Comment (HydroGames-dev) [reply src/alvin0319/AmongUs/game/Game.php]: k

**Pull request merged**